product_docs/docs/epas/11/installing/windows_install_details/05_managing_an_advanced_server_installation/index.mdx

```diff
@@ -1,5 +1,7 @@
 ---
 title: "Managing an Advanced Server Installation"
+redirects:
+ - /epas/11/epas_inst_windows/05_managing_an_advanced_server_installation/
```

Comment:
This comment is on this file, not on this particular bullet.

Can we pull all the subtopics here to make it consistent with the same topic of Linux? Also, the title here 'Starting and stopping..." is for all the subtopics and not only for the content here. See "Managing Advanced Server installation" topic of Linux.

Reply:
Are you saying we should put all the topics that are subordinate to "Managing Advanced Server installation" into the same file? It seems like Linux does something like that.

As for "starting and stopping", I have a similar question. Are you saying put all the topics subordinate to "Managing an Advanced Server installation" into the same topic level topic.

Reply:
Yes